roles/network_plugin/calico/defaults/main.yml

```diff
@@ -155,3 +155,6 @@ calico_ipam_maxblocksperhost: 0
 
 # Calico apiserver (only with kdd)
 calico_apiserver_enabled: false
+
+# calico feature detect override
```

Comment:
Should we add some comments:
```
Calico feature detect override, set  "ChecksumOffloadBroken=true"  to solve the https://github.com/projectcalico/calico/issues/3145 .
```

like the https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/main/templates/addons/calico/patches/calico-node.yaml , so that the user can know how to set it.

Reply:
updated